src/webhook/test/unit/webhook.service.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { AppModule } from '../../../app.module';
+import axios from 'axios';
+import { RecipeResponse } from '../../../recipe/responses/recipe.response';
+import { WebhookService } from '../../webhook.service';
+
+const recipeForWebhook: RecipeResponse = {
+  id: 1,
+  name: 'testName',
+  description: 'testDescription',
+  fileId: 1,
+  authorId: 1,
+  preparing: [
+    {
+      step: 'testStep',
+      order: 1,
+    },
+  ],
+  ingredients: [
+    {
+      name: 'testName',
+      amount: 'testAmout',
+    },
+  ],
+  fileUrl: 'testFileUrl',
+};
+
+describe('Webhook Service', () => {
+  let webhookService: WebhookService;
+
+  beforeAll(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    await module.createNestApplication().init();
```

Comment:
Did you mocked the axios/http service? I think you are making real requests inside tests. 
Http request should not be sent in tests